packages/ffe-radio-button-react/src/RadioButtonGroup.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { bool, func, oneOf, shape, string } from 'prop-types';
+import { Tooltip } from '@sb1/ffe-form-react';
+
+class RadioButtonGroup extends Component {
+    render() {
+        const {
+            'aria-invalid': ariaInvalid,
+            children,
+            inline,
+            description,
+            name,
+            onChange,
+            selectedValue,
+            tooltip,
+            tooltipProps,
+        } = this.props;
+
+        const buttonProps = { 
+            'aria-invalid': ariaInvalid, 
+            inline,
+            name,
+            onChange,
+            selectedValue,
+        };
+
+        return (
+            <fieldset 
+                className="ffe-fieldset" 
+                onChange={this.onChange}
```

Comment:
This should be `onChange` (sans `this`), right? To use the prop.

Reply:
Very true - this was some left-behind garbage from a previous iteration. Nicely spotted!